sources/System/BombHolder/BombHolderSystem.cpp

```diff
@@ -189,7 +189,7 @@ namespace BBM
 		}
 		if (controllable.bomb && holder.bombCount > 0) {
 			auto spawnPos = position.position.round();
-			for (auto &[entity, pos, _] : this->_wal.getScene()->view<PositionComponent, BasicBombComponent>()) {
+			for (auto &[__, pos, _] : this->_wal.getScene()->view<PositionComponent, BasicBombComponent>()) {
```

Comment:
__ is a reserved name in C and C++. The compiler has the right to define this to anything without calling it a breaking change. We should not have that in the code.